slam-ws/src/nav_husky_gazebo/launch/move_base_amcl.launch

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
```

Comment:
Call `move_base.launch` from within this `.launch` file and specify non-default parameters/arguments when calling `move_base.launch` (so it can use the amcl config files)